source/fundamentals/crud/read-operations/retrieve.txt

```diff
@@ -0,0 +1,121 @@
+==============
+Retrieve Data
+==============
+
+.. default-domain:: mongodb
+
+
+Read operations allow you to retrieve data from your MongoDB database.
+
+You can retrieve data using find, aggregate, and watch operations.
+
+A find operation is useful if you want a subset of the existing data
+based on your query filter. You can further specify which documents to
+retrieve such as how to sort results, skip results, limit results and
+limit the fields to return. 
+
+An aggregate operation is useful if you want to perform an ordered
+pipeline of transformations to the matched data. You can perform all
+find operations, rename fields, calculate fields, summarize data and
+group values.
+
+A watch operation is useful if you want to monitor the database in
+real-time. You can specify which write operations to monitor such as
+insert, update and delete.
```

Comment:
Maybe instead of organizing this section by features, it can be organized by what actions a reader may want to perform. It maybe could be a list of use cases. For example:

You should read this guide if you would like to:
- Retrieve a subset of your database
- Perform transformations on a subset of your database (this is weak, but I think captures my meaning)
- Monitor changes to your database